ansible/roles/ocp4-workload-security-compliance-lab/tasks/infrastructure.yml

```diff
@@ -1,103 +1,104 @@
 ---
-- name: infrastructure Tasks Started
-  debug:
-    msg: "infrastructure Tasks - Started"
-
-#####
-#
-- name: Install OpenSCAP tools
-  yum:
-    name:
-      - docker 
-      - openscap-utils
-      - openscap-scanner
-      - scap-security-guide
-      - atomic
-  delegate_to: "{{ bastion_internal }}"
-
-- name: Ensure docker group exists
+
+# Workaround for Python3 incompatibility
+- name: install oscap tools
+  raw: yum install -y docker openscap-utils openscap-scanner scap-security-guide atomic
```

Comment:
look at this to see if you can use RHEL 8 and podman, if not we need to override bastion image in config to use RHEL 7

Reply:
This will be moved to RHEL 8 in a smaller future PR.